### PR TITLE
405: lighthouse score improvements

### LINKED ISSUE #405: Score at least 90 for all Google Lighthouse scores.
Run Lighthouse in your developer tools and some issues are listed that need to be fixed to improve the SEO, usability, quality and performance of your website.

Screenshot 2021-11-13 at 05 46 31

### PR DESCRIPTION
**Pull request by piersss**
Closes #405 

Mostly some low hanging fruit.

- Accessibility and Best Practices are in the green now
- Not able to fix the main Performance issues, one is caused by typekit and one is something about React.lazy(). Was not able to fix this.
- Removed unused DarkModeSwitch component

You're welcome to make commits to this branch if you find any way to improve scores.

### CODE REVIEW

File: src/components/SwapWidget/InfoSection.tsx [hunk 1]
```diff
@@ -189,6 +190,7 @@ const InfoSection: FC<InfoSectionProps> = ({
             <FeeText>{t("marketing:includesFee")}</FeeText>
             <InfoButton
               onClick={onFeeButtonClick}
+              ariaLabel="More information"
```
Comment (robbestad) [inline src/components/SwapWidget/InfoSection.tsx:193]: use translations for these

**Review by robbestad — CHANGES_REQUESTED** [review]
I commented only one place, but use translations for all ariaLabels.

File: src/components/SwapWidget/InfoSection.tsx (hunk 1 above)
Comment (piersss) [reply src/components/SwapWidget/InfoSection.tsx:193]: Whoopsy!

**Comment by robbestad** [comment]
Very nice that you added keys for nb locale! Do you mind if I just change the wording for a couple of them by pushing onto this branch?

**Comment by piersss** [comment]
> Very nice that you added keys for nb locale! Do you mind if I just change the wording for a couple of them by pushing onto this branch?

Yeah sure!

**Review by robbestad — APPROVED** [review]

**Pull request merged**